### ISSUE TITLE
Import tracking (ES6, webpack)

### ISSUE THREAD

**Issue by nosy-b** [eduardolundgren/tracking.js#177]
Thanks for this great library.

I wanted to use it in a web project built using webpack. In the package.json i added 

> "tracking": "^1.1.2" 

which works fine, it's built and goes into node_modules.
But how do you use it in a class? `import tracking from 'tracking';` doesn't work, nor the Require version (even with the build tracking.js placed into the sources)
Thank you

**Comment by SimoneS93** [comment]
I don't know about webpack, but I edited the tracking.js file to make it work on node with require('./tracking'). Here's it: https://github.com/SimoneS93/trackingjs

**Comment by psigen** [comment]
Having a similar problem here.  I have installed the package via `npm`, but it is impossible to include directly using `import` or `require` syntax.

It would be great if this module followed any of the JS module standards, but the most useful would probably be [ES6 modules](https://github.com/nzakas/understandinges6/blob/master/manuscript/13-Modules.md).

As @SimoneS93 noted, this is as simple as adding a `module.exports` (CommonJS) or `export` (ES6) statement to the end of the library.

**Comment by samburgers** [comment]
Looks like they take pull requests ^^

**Comment by geekplux** [comment]
you can try to use webpack shimming:

```js
rules: [{
  test: require.resolve('tracking'),
  use: 'imports-loader?this=>window
}]

...

plugins: [
  new webpack.ProvidePlugin({
    tracking: 'tracking'
  })
]
```

**Comment by joZephhh** [comment]
is that currently supports ? :)

**Comment by jsprow** [comment]
Hopefully this'll help someone.

I was running into the same problem importing tracking.js, and went the Webpack shim route as suggested by @geekplux , copying the code exactly as written, after installing the files via `npm install tracking --save`.

When I imported `tracking`, I found I also needed to import the libraries for `ObjectTracker`. I thought I'd need to dive back into `webpack.config.js`,  but it worked just fine using: 

```js
import tracking from 'tracking';
import 'tracking/build/data/face-min.js';
import 'tracking/build/data/eye-min.js';
import 'tracking/build/data/mouth-min.js';
```
